Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -326,7 +326,7 @@ public void onAdStart(@NonNull String placementReferenceId) {
                     @Override
                     public void run() {
                         if (mInteractionListener != null) {
-                            mInteractionListener.onAdImpression();
+                            mInteractionListener.onAdShown();
```

Comment:
I'm unsure about this one though. We never sent onAdShown from Banners, so maybe I should just delete this line and surroundings.

Reply:
We must call onAdDismissed when onAdEnd if onAdShown is called here.

Reply:
could be in onAdEnd

Reply:
@anton-blashchuk-zoolatech we can remove onAdShown for Banners, from my understanding its related to the separate show() call for Interstitial/Rewarded. For Banners as we dont have show, the shown/dismissed events are not relevant
`
/**
     * Implementers should now show the ad for this base ad. Optional for inline ads that correctly
     * return a view from getAdView
     */
    protected void show()
`